src/shared/components/Animations/ListItem.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import {
+  withKnobs,
+  text,
+  array,
+  object,
+  boolean,
+} from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import ListItem from './ListItem';
+import { Tag, Button } from 'antd';
+
+const exampleItems = [
+  {
+    label: 'nexus',
+    projectNumber: 1200,
```

Comment:
Do we stick with that `projectNumber` thing? I though we scrapped that?

Reply:
just an arbitrary example